vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj.filters

```diff
@@ -313,7 +313,9 @@
     <ClCompile Include="JSI\ChakraJsiRuntime_edgemode.cpp">
       <Filter>JSI</Filter>
     </ClCompile>
-    <ClCompile Include="DesktopWindowMessage.cpp" />
```

Comment:
I need to look at why VS removed this...

Reply:
this is benign, when I converted this type from runtimeclass to struct these files went away but it seems like I didn't commit the changes to the vcxproj